mmedit/apis/video_interpolation_inference.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os
+import os.path as osp
+
+import mmcv
+import numpy as np
+import torch
+from mmcv.fileio import FileClient
+from mmcv.parallel import collate, scatter
+
+from mmedit.core import tensor2img
+from mmedit.datasets.pipelines import Compose
+
+VIDEO_EXTENSIONS = ('.mp4', '.mov', '.avi')
+FILE_CLIENT = FileClient('disk')
+
+
+def read_image(filepath):
+    """Read image from file.
+
+    Args:
+        filepath (str): File path.
+
+    Returns:
+        image (np.array): Image.
+    """
+    img_bytes = FILE_CLIENT.get(filepath)
+    image = mmcv.imfrombytes(
+        img_bytes, flag='color', channel_order='rgb', backend='pillow')
+    return image
+
+
+def video_interpolation_inference(model,
+                                  input_dir,
+                                  start_idx=0,
+                                  end_idx=None,
+                                  batch_size=4):
+    """Inference image with the model.
+
+    Args:
+        model (nn.Module): The loaded model.
+        input_dir (str): Directory of the input video.
+        start_idx (int): The index corresponds to the first frame in the
+            sequence. Default: 0.
+        end_idx (int | None): The index corresponds to the last interpolated
+            frame in the sequence. If it is None, interpolate to the last
+            frame of video or sequence. Default: None.
+        batch_size (int): Batch size. Default: 4.
```

Comment:
Although it is not a major issue, it's better to be consistent with the default value in `demo`.